app/components/Footer/style.module.css

```diff
@@ -0,0 +1,73 @@
+.footer {
+  background: var(--gradiente-light);
+  word-break: break-word;
+  padding-top: 20rem;
+  margin-top: -16rem;
+  z-index: 0;
```

Comment:
Qual a utilidade do z-index 0 aqui?

Reply:
Foi um pedido do Ronan para ser feita a integração da seção do Linked-in com o footer

Reply:
@ronansalvador z-index aqui tem alguma utilidade?

Reply:
eu sugeri pois quando eu tentei fazer localmente a junção com o linkedin estava sobrepondo

Reply:
Vamos manter, mas vamos ficar de olho quando vier linkedin